src/Hint/Lambda.hs

```diff
@@ -142,6 +143,7 @@ lambdaExp p o@(Lambda _ pats x) | isLambda (fromParen x), null (universeBi pats
       munge ident p@(PWildCard _) = p
       munge ident p = PVar (ann p) (Ident (ann p) [ident])
       subts = ("body", toSS body) : zipWith (\x y -> ([x],y)) ['a'..'z'] (map toSS pats)
+lambdaExp p o@(Lambda _ [view -> PVar_ u] (Case _ (view -> Var_ v) alts)) | u == v = [suggestN "Use λ-case" o $ LCase an alts]
```

Comment:
This goes wrong if u is reused in one of the alts. That case needs guarding for.

Reply:
Can we stick to ascii both in the source and console messages, so lambda-case.